spec/models/billable_metric_spec.rb

```diff
@@ -0,0 +1,31 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe BillableMetric, type: :model do
+  subject(:billable_metric) { BillableMetric.new }
```

Comment:
RSpec/DescribedClass: Use `described_class` instead of `BillableMetric`. (https://www.rubydoc.info/gems/rubocop-rspec/RuboCop/Cop/RSpec/DescribedClass)